README.md

```diff
@@ -108,7 +108,7 @@ for an example workspace using another scala version.
 
 | bazel  | rules_scala gitsha |
 |--------|--------------------|
-| 1.1.0  | HEAD               |
+| 1.2.1  | HEAD               |
 | 0.28.0 | HEAD               |
```

Comment:
It looks like we are no longer testing with an older version (we used to test with the oldest and the newest that worked and hoped all the ones in the middle would work).

We should probably find the sha that we switched away from 0.28.0 and put the label here or remove this table.

Since we don't test the oldest that works we don't really know what versions work for which shas, so if we are just going to test with 1, we should probably remove the table (since it isn't very accurate).

Reply:
I am fine either way as I am always using the latest though.

Reply:
Another thing I can think of is to introduce more testing in the bazelci and it would cover more platforms.

Reply:
It's hard for many orgs to update large builds with many rules not all of which support the latest versions. It is much more of a service to tell users the versions that work for each version of bazel rather than them having to git-bisect it.

Reply:
yeah, that is why I think this sha version table might be still quite useful though.

Reply:
I will search the SHA for 0.28.0 and update that sha column.

Reply:
without identifying the first failing commit, how do we know what sha to put in the table?

Reply:
I used the README as first known good hash for the v0.28.0

Reply:
Actually, I've totally been misleading us...

HEAD still works for 0.28.1 which is the default version. We are testing twice. We should probably be explicit rather than one with and without a version, just be explicit about the versions so we don't have this confusion...

So, I think you can keep HEAD as the 0.28 version (which is nice that we work for such a wide range of versions). Sorry for the noise, but I guess this also means we should comment the test script a bit better since I even forgot about this (after having reviewed the last change to this script).

Reply:
Sure, I will revert my 0.28.0 HEAD change.

Reply:
+1 to explicitly labeling the default commit.
@chenrui333 Id love for you to do this here but you can feel free to do it in a separate PR or not at all and I’ll do it.
Thanks!

Reply:
> +1 to explicitly labeling the default commit.

@ittaiz I guess I missed on the default commit, do you prefer HEAD or you prefer explicit SHA?

Reply:
This is great. I miswrote. I meant labeling in the Travis config and the test script the default Bazel version (not commit).
I’ll merge this as is.
If you can follow up with a PR with this I’d be grateful :)

Reply:
Sure, I can file a follow up PR on this.